### DISCUSSION TITLE
Add edge cases guide to `<Drawer>` docs

### DISCUSSION THREAD

**Discussion by bitobeats** [corvudev/corvu#26] [Ideas]
The `<Drawer>` component uses the `onPointerDown` event to handle dragging.

I found an edge case (symptom is the same as the one describe in [this issue](https://github.com/corvudev/corvu/issues/18)) where clicking `<select>` inside a Drawer and then clicking outside keeps the dragging state true. It happens only in Safari as far as I tested, and presumably improperly fires the `onPointerDown` event in `<Drawer>` .

It can be avoided by doing `<select onPointerDown={(ev) => ev.stopPropagation}>`, so it's not a big problem.

I have a feeling this is not a corvu bug; instead, something that should be dealt with granularly by the user.

So I'm opening this discussion to suggest that the docs mentions that edge cases similar to this one can be easily avoided by adding `ev.stopPropagation()` to the target's `onPointerDown` event.

**Comment by GiyoMoon** [comment]
Good catch! This is actually fixable, corvu can check if the target element is a `<select>` element and abort the pointer down event. I'll release a fix shortly!

**Reply by GiyoMoon** [reply]
Additionally, if you are dealing with any kind of elements where you want to disable dragging, you can add the `data-corvu-no-drag` attribute and corvu will ignore any events from these elements.
Docs: https://corvu.dev/docs/primitives/drawer/#disable-dragging-on-certain-elements

**Comment by GiyoMoon** [comment]
A fix is released as `@corvu/drawer@0.2.1`. [Release notes](https://github.com/corvudev/corvu/releases/tag/%40corvu%2Fdrawer%400.2.1)
